content/en/docs/reference/kubectl/cheatsheet.md

```diff
@@ -390,6 +390,7 @@ Verbosity | Description
 `--v=2` | Useful steady state information about the service and important log messages that may correlate to significant changes in the system. This is the recommended default log level for most systems.
 `--v=3` | Extended information about changes.
 `--v=4` | Debug level verbosity.
+`--v=5` | Trace level verbosity.
```

Comment:
How to know this level is Trace?

Reply:
@oomichi @sangam14
See:
https://github.com/kubernetes/community/blob/4eeba4d57bed01502cb09598a74d21671d4ee876/contributors/devel/sig-instrumentation/logging.md#L28
(not sure why GitHub isn't displaying contents 🙄)

> klog.V(5) - Trace level verbosity

Reply:
`kubectl` uses `klog`.

Reply:
klog have flags from --v=0 to 5 . and 4 is missing from  official documentation !

Reply:
@sangam14 I'm not sure what you mean&mdash;level for is documented at https://github.com/kubernetes/community/blob/4eeba4d57bed01502cb09598a74d21671d4ee876/contributors/devel/sig-instrumentation/logging.md#L28.

Reply:
@bl-ue 
Thanks for your reply, that is a perfect answer for me :-)

/lgtm